common/utils/board.ts

```diff
@@ -154,12 +151,14 @@ export function applySingleUse(
 
 	currentPlayer.board.singleUseCardUsed = true
 
-	game.battleLog.addApplyEffectEntry(effectAction)
 	currentPlayer.hooks.onApply.call()
 
 	// This can only be done once per turn
 	game.addCompletedActions('PLAY_SINGLE_USE_CARD')
 
+	// Send the logs
```

Comment:
Comment is send the logs but this isn't sending the logs, is it?

Reply:
Yeah, typo